### PR TITLE
Change exercise view

### PR DESCRIPTION
**Pull request by Christen97**
Now the admins get redirected to the task instead of the analytics and can access the analytics from there.

### CODE REVIEW

File: resources/views/tasks/show.blade.php [hunk 1]
```diff
@@ -2,9 +2,19 @@
 
 @section('content')
     <task :task="{{ $task }}"
+          @if(auth()->user()->is_admin ==  1)
```
Comment (mcamp) [inline resources/views/tasks/show.blade.php]: Please refactor this now before it gets more complicated in the future. Use boolean values

**Comment by mcamp** [comment]
check also the conflict, failed and warning tests

File: resources/views/tasks/show.blade.php [hunk 2]
```diff
@@ -2,9 +2,19 @@
 
 @section('content')
     <task :task="{{ $task }}"
+          @can('viewAnalytics', $task)
+              edit-route="{{ route('courses.tasks.admin.preferences', [$course, $task]) }}"
+          @else
+              edit-route="{{ null }}"
```
Comment (jazerix) [inline resources/views/tasks/show.blade.php]: Is the else actually necesarry? In the Task.vue, what does the prop evaluate to, if it's not passed through this view?

Comment (Christen97) [reply resources/views/tasks/show.blade.php]: If I dont pass null through the buttton is visible for non admins. not sure how I can tell what the prop evaluates to if its not passed through.

**Pull request merged**